### DISCUSSION TITLE
捐赠致谢初始记录名单

### DISCUSSION THREAD

**Discussion by Mq-b** [Mq-b/Modern-Cpp-templates-tutorial#5] [Show and tell]
re早期其他捐赠不计入本名单；本名单只用来暂时记录，后续单独写界面致谢。

1. [**golezi**](https://github.com/golezi/) 捐赠：**20**
2. [**mq五彩斑斓**](https://github.com/DataEraserC/) 捐赠：**20**
3. **edf282c6e3** 捐赠：**10**
4. [**mq卢瑟**](https://github.com/mq-loser) 捐赠：**1774** 上贡：“`rk68`”、“`VGN S99`”
5. [**akchilov**](https://github.com/AzrBrk) 捐赠：**20**
6. [**心洗**](https://github.com/MrShinshi) 上贡：**512**
7. [**mq松鼠**](https://github.com/CSTGluigi) 捐赠：**50**
8. [**mq红**](https://github.com/somniumchase) 捐赠：**50**
9. [**$_0x00f0**](https://github.com/S-0xff0f) 捐赠：**睿站季度大会员（68）**
10. [**涼風青葉**](https://github.com/Suzukaze7) 上贡：**80**
11. [**mq星**]() 捐赠：**10**
12. [**mq眠**]() 捐赠：**540**
13. **神秘人1** 捐赠：**300**
14. **神秘人2** 捐赠：**10**
15. [**hypatiy**]() 捐赠：**2272**
16. [**mq舒**]() 捐赠：**70**
17. [**Yuzhiy05**]() 捐赠：**700**（大约）
18. [**MSC☆Xiphoillumination**]() 捐赠：**1853**（大约）
19. [**Da'Inihlus**]() 捐赠：**50**（大约）
20. [**拉达姆**]() 捐赠：**7**
21. [**mq日**]() 捐赠：**8.8**
22. [**日向花音**]() 给零花钱：**133.3**
23. [**Schumacher RyoSukE**]() 赞助：**33.3**
24. [**等疾风**](https://github.com/Codesire-Deng) 赞助：**50**
25. [**白月光**]() 赞助：**6.66**。
26. **不愿透露姓名的神秘人** 赞助：**500**
27. [**PantheraLeo**](https://github.com/PantheraLeo14) 赞助：**45**
28. [**噗叽**](https://github.com/puji4810) 纳税：**53**
29. [**水夕**](https://github.com/sakria9) 上贡：**149**
30. [**叠别拷打我**](https://github.com/Yuria-Shikibe) 赞助：**88.88**
31. [**群饿龙🍔🍟🍹🍜🍗🥖🎂🥗**](https://github.com/StinkyTooFool) 赞助：**157.68**
32. [**寻宝游戏**]() 赞助：**197.76**
33. [**我是男孩**](https://github.com/zhuzhu9) 上贡：**230** + 1年svip + 3000人群。
34. [**卖女孩的小火柴**](https://github.com/weasyd) 捐赠：**10**
35. [**saidfljwnzjasf**](https://github.com/KSARK) 捐赠：**118.88**
36. [**懒狗**](https://github.com/RaDsZ2z) 上贡：**50**
37. [**林木甜**]() 上贡：**21.3**
38. [南花弄影墨](https://github.com/Dwc0319) 捐赠：**30**
39. [Origin]() 捐赠：**66.66**
40. 风云雾客 捐赠：**70**

早期其实还有非常之多，多早呢，刚做视频的时候，有些人就是直接加我qq就给我转钱，我不知道谁是谁，以及还有一些大赞助行为，是早期，我也没记录，暂时就这样吧，有遗漏记得补充。

- 神秘人：指不知道是谁，直接转钱，比如通过支付宝，询问无果。
- 不愿透露姓名的神秘人：指知道是谁，不愿透露

## 此表单不再更新

致谢页面已经完成，转到：https://mq-b.github.io/Modern-Cpp-templates-tutorial/image/%E6%8D%90%E8%B5%A0/thanks

**Comment by MrShinshi** [comment]
![81587338](https://github.com/Mq-b/Modern-Cpp-templates-tutorial/assets/103853130/06a0ff3b-c0b2-43f9-9b5a-e4b16310b443)
太好了,国王心里没我

**Reply by MrShinshi** [reply]
我一点也不难过,真的555

**Reply by Mq-b** [reply]
好好好好好
